src/Widgets/calendar/ControlHeader.vala

```diff
@@ -25,28 +25,51 @@ namespace DateTime.Widgets {
         public signal void center_clicked ();
         public ControlHeader () {
             Object (orientation : Gtk.Orientation.HORIZONTAL);
-            var left_button = new Gtk.Button.from_icon_name ("pan-start-symbolic");
-            var right_button = new Gtk.Button.from_icon_name ("pan-end-symbolic");
-            var center_button = new Gtk.Button.with_label (new GLib.DateTime.now_local ().format (_("%OB %Y")));
+
+            var label = new Gtk.Label (new GLib.DateTime.now_local ().format (_("%OB, %Y")));
+            label.get_style_context ().add_class (Granite.STYLE_CLASS_H2_LABEL);
+            label.halign = Gtk.Align.START;
+            label.set_size_request (185, 24);
+            label.xalign = 0;
+
+            var left_button = new Gtk.Button.from_icon_name ("go-previous-symbolic");
```

Comment:
I think in this context, `pan-start-symbolic` and `pan-end-symbolic` probably still look nicer